crates/turbopack-build/js/src/runtime.js

```diff
@@ -0,0 +1,359 @@
+/* eslint-disable @next/next/no-assign-module-variable */
```

Comment:
This is a slimmed down version of our dev runtime that doesn't include HMR.

It's currently missing some more recent updates to our dev runtime. 

Eventually, this runtime should be optimized for conciseness and performance. But for now, readability is privileged.